fossunited/chapters/doctype/foss_chapter_events/foss_chapter_events.py

```diff
@@ -1,9 +1,15 @@
 # Copyright (c) 2023, Frappe x FOSSUnited and contributors
 # For license information, please see license.txt
 
-# import frappe
+import frappe
 from frappe.website.website_generator import WebsiteGenerator
 
 
 class FOSSChapterEvents(WebsiteGenerator):
-    pass
+    def after_insert(self):
+        events = frappe.get_doc("FOSS Chapter Events")
+        print(events)
+        if self.status == "Approved":
+            new_list = frappe.new_doc("Email Group")
+            new_list.title = self.event_name
+            new_list.insert()
```

Comment:
Why add this now?

Reply:
That got pushed mistakenly, I have reverted to default